### PR TITLE
Added note on TDE presence check

### PR DESCRIPTION
**Pull request by djw-m**

### CODE REVIEW

File: product_docs/docs/tde/15/enabling_tde.mdx [hunk 1]
```diff
@@ -65,7 +65,8 @@ If you don't specify this parameter, the environment variable `PGDATAKEYUNWRAPCM
 
 Use the special value `-` if you don't want to apply any key unwrapping command.
 
-<!--Should this be worded differently? Is this the environment variable fallback? I changed the word "option" to parameter. -->
+<!--- Should this be worded differently?
```
Comment (nidhibhammar) [inline product_docs/docs/tde/15/enabling_tde.mdx:68]: I think we will be removing this and the following line

File: product_docs/docs/tde/15/enabling_tde.mdx [hunk 2]
```diff
@@ -108,3 +109,17 @@ This example uses EDB Postgres Advanced Server 15 running on a Linux platform. I
    data_encryption_key_unwrap_command = 'openssl enc -d -aes-128-cbc -pass pass:ok -in %p'
    ```
 
+## Checking for TDE in use using SQL
```
Comment (nidhibhammar) [inline product_docs/docs/tde/15/enabling_tde.mdx:112]: Can we reword this like - Checking for TDE presence using SQL. or something else? Intend is not to use "use" along with using.

**Review by nidhibhammar — CHANGES_REQUESTED** [review]
Rest looks good to me

File: product_docs/docs/tde/15/enabling_tde.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/tde/15/enabling_tde.mdx]: Removed... No idea how the line break appeared, but we should resolve questions about content without leaving long lived and untracked comments in the docs.

File: product_docs/docs/tde/15/enabling_tde.mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/tde/15/enabling_tde.mdx]: Done

File: product_docs/docs/tde/15/enabling_tde.mdx [hunk 3]
```diff
@@ -108,3 +106,17 @@ This example uses EDB Postgres Advanced Server 15 running on a Linux platform. I
    data_encryption_key_unwrap_command = 'openssl enc -d -aes-128-cbc -pass pass:ok -in %p'
    ```
 
+## Checking for TDE presence using SQL
+
+The presence of TDE on a server can be determined by querying the `data_encryption_version` column of the `pg_control_init` table. 
+
+A value of 0 means TDE is not enabled. Any non-zero value reflects the version of TDE in use. Currently, when TDE is enabled, this value will be 1.
```
Comment (drothery-edb) [inline product_docs/docs/tde/15/enabling_tde.mdx:113]:
```suggestion
A value of 0 means TDE isn't enabled. Any non-zero value reflects the version of TDE in use. Currently, when TDE is enabled, this value is 1.
```

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**